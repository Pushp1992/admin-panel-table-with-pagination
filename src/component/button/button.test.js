import {render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Button from './button';
import '@testing-library/jest-dom';

afterEach(() => {
    cleanup()
});

it('matches the snapshot with actual DOM tree', () => {
    const component = renderer.create(<Button/>);
    const DOMTree = component.toJSON();
    expect(DOMTree).toMatchSnapshot();
});

it('renders Button component', () => {
    render(<Button/>);
    const ButtonElement = screen.getAllByTestId('button');
    expect(ButtonElement[0]).toBeInTheDocument();
})