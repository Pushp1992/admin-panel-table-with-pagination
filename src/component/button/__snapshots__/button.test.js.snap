// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot with actual DOM tree 1`] = `
<button
  data-testid="button"
/>
`;
